src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjectsTable.tsx

```diff
@@ -0,0 +1,135 @@
+      density="compact"
+      initialState={{
+        sorting: {
+          sortModel: [{ field: 'name', sort: 'desc' }],
+        },
+      }}
+      disableColumnMenu
+      disableSelectionOnClick
+      autoHeight
+      hideFooter={true}
+      showCellRightBorder={false}
+      rowSpacingType="margin"
+      sx={{
+        border: 'none',
+        '& .MuiDataGrid-columnSeparator': {
+          display: 'none',
+        },
+        '& .MuiDataGrid-columnHeaders': {
+          borderBottom: 'none',
+        },
+        '& .MuiDataGrid-columnHeaderTitle': {
+          fontWeight: 'bold',
+        },
+        '& .MuiDataGrid-cell': {
+          borderBottom: 'none',
+        },
+        '& .MuiDataGrid-row:hover': { cursor: 'pointer' },
+        '& .MuiDataGrid-cell, & .MuiDataGrid-columnHeader': {
+          '&:focus, &:focus-within': { outline: 'none' },
+        },
+        '& .MuiDataGrid-columnHeader:nth-last-of-type(-n+2) .MuiDataGrid-columnSeparator--sideRight':
+          {
+            display: 'none',
+          },
+      }}
+      rows={projects}
+      onRowClick={handleRowClick}
+      columns={[
+        {
+          headerName: 'Project Name',
+          field: 'name',
+          flex: 1,
+          renderCell: (params) => {
+            const nameObj = params.row.name;
+            return (
+              <span style={{ color: theme.palette.primary.main }}>
+                {nameObj.value ?? ''}
+              </span>
+            );
+          },
+          sortingOrder: ['desc', 'asc'],
+        },
+        {
+          headerName: 'Status',
+          field: 'status',
+          flex: 1,
+          renderCell: (params) => {
+            const status = params.row.projectStatus;
+            return ProjectStatusLabels[status];
+          },
+          sortingOrder: ['desc', 'asc'],
+        },
+        {
+          headerName: 'Engagements',
+          field: 'engagements',
+          flex: 1,
+          renderCell: (params) => {
+            return params.row.engagements.total;
+          },
+          sortingOrder: ['desc', 'asc'],
+        },
+        {
+          headerName: 'Sensitivity',
+          field: 'sensitivity',
+          flex: 1,
+          renderCell: (params) => {
+            let color;
+            switch (params.row.sensitivity.toLowerCase()) {
+              case 'high':
+                color = theme.palette.error.main;
+                break;
+              case 'medium':
+                color = theme.palette.warning.main;
+                break;
+              case 'low':
+                color = theme.palette.primary.main;
+                break;
+              default:
+                color = 'inherit';
+            }
+
+            return (
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <ShieldIcon style={{ color: color, marginRight: '5px' }} />
+                {params.row.sensitivity.toUpperCase()}
+              </div>
+            );
+          },
```

Comment:
We have a component that abstracts all of the icon stuff.
```suggestion
          renderCell: ({ value }) => (
            <Box
              display="flex"
              alignItems="center"
              gap={1}
              textTransform="uppercase"
            >
              <SensitivityIcon value={value} disableTooltip />
              {value}
            </Box>
          ),
```